### PR TITLE
raplace the given lines of code with <Formattedstring /> later find more

### PR DESCRIPTION
**Pull request by sparkingdark**
I am done with given links of code lines and now will do some more searching but first please check this task.
issue #716

### CODE REVIEW

File: src/app/components/media/CreateRelatedMediaDialog.js [hunk 1]
```diff
@@ -97,7 +97,7 @@ class CreateRelatedMediaDialog extends React.Component {
         </DialogContent>
         <DialogActions>
           <Button id="create-media-dialog__dismiss-button" onClick={this.props.onDismiss}>
-            {this.props.intl.formatMessage(globalStrings.cancel)}
+            {<FormattedMessage {...globalStrings.cancel} />}
```
Comment (amoedoamorim) [inline src/app/components/media/CreateRelatedMediaDialog.js:100]: Please remove outer curly braces as they're unnecessary here

File: src/app/components/task/EditTaskDialog.js [hunk 2]
```diff
@@ -238,13 +238,13 @@ class EditTaskDialog extends React.Component {
         ))}
         <div style={{ marginTop: units(1) }}>
           <Button onClick={this.handleAddValue.bind(this)}>
-            {this.props.intl.formatMessage(messages.addValue)}
+            {<FormattedMessage {...messages.addValue} />}
```
Comment (amoedoamorim) [inline src/app/components/task/EditTaskDialog.js:241]: Please remove outer curly braces as they're unnecessary here.
Also please add `id` and `defaultMessage` as props to the `FormattedMessage` component directly and remove the `addValue` key from `messages` object.

File: src/app/components/task/EditTaskDialog.js [hunk 3]
```diff
@@ -238,13 +238,13 @@ class EditTaskDialog extends React.Component {
...
           </Button>
           <Button
             onClick={this.handleAddOther.bind(this)}
             disabled={this.state.hasOther}
           >
-            {this.props.intl.formatMessage(messages.addOther)}
+            {<FormattedMessage {...messages.addOther} />}
```
Comment (amoedoamorim) [inline src/app/components/task/EditTaskDialog.js:247]: Please remove outer curly braces as they're unnecessary here.
Also please add `id` and `defaultMessage` as props to the `FormattedMessage` component directly and remove the `addOther` key from `messages` object.

**Review by amoedoamorim — CHANGES_REQUESTED** [review]
Thank you @sparkingdark for your contribution! We appreciate it! :muscle: 
In order for your PR to be accepted and merged please apply these requested changes.
Again, thanks for participating!

**Comment by sparkingdark** [comment]
Okay Doing it

File: src/app/components/media/CreateRelatedMediaDialog.js (hunk 1 above)
Comment (sparkingdark) [reply src/app/components/media/CreateRelatedMediaDialog.js]: please check once I add it but what will be the default message I wrote defaultMessage="props".

File: src/app/components/task/EditTaskDialog.js [hunk 4]
```diff
@@ -238,13 +238,13 @@ class EditTaskDialog extends React.Component {
...
           <Button onClick={this.handleAddValue.bind(this)}>
-            {this.props.intl.formatMessage(messages.addValue)}
+            <FormattedMessage id="props" defaultMessage="props" {...messages} />
           </Button>
           <Button
             onClick={this.handleAddOther.bind(this)}
             disabled={this.state.hasOther}
           >
-            {this.props.intl.formatMessage(messages.addOther)}
+            <FormattedMessage id="props" defaultMessage="props" {...messages} />
```
Comment (amoedoamorim) [inline src/app/components/task/EditTaskDialog.js:247]: Same here, `id` and `defaultMessage` should not be the `"props"` string. They should have the actual values that are in `messages.addOther`.

**Review by amoedoamorim — CHANGES_REQUESTED** [review]

**Comment by amoedoamorim** [comment]
Hello @sparkingdark, are you still around? Please let me know if you're still interested in fixing this!

**Comment by sparkingdark** [comment]
Yeah I am doing it.

**Comment by amoedoamorim** [comment]
Please update your branch with latest `develop` and watch out for conflicts

**Pull request closed without merging**